app.js

```diff
@@ -2,13 +2,13 @@ const express = require('express');
 const app = express();
 const {
     getApi,
-    // getTopics
+    getTopics
 } = require('./controllers/app.controller');
 
 app.use(express.json());
```

Comment:
Is this necessary yet? We shouldn't be committing code we don't need. What would you want app.use(express.json()) for? 🤔